.github/workflows/tests-evm.yml

```diff
@@ -11,13 +11,68 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
   cancel-in-progress: true
 
+permissions:
+  pull-requests: write
+
 jobs:
   isdraft:
     uses: ./.github/workflows/reusable-isdraft.yml
   preflight:
     needs: isdraft
     uses: ./.github/workflows/reusable-preflight.yml
 
+  differential-tests-revive-dev-node-revm:
+    needs: [preflight]
+    runs-on: ${{ needs.preflight.outputs.RUNNER }}
+    if: ${{ needs.preflight.outputs.changes_rust }}
+    timeout-minutes: 60
+    container:
+      image: ${{ needs.preflight.outputs.IMAGE }}
+    steps:
+      - name: Checkout the Polkadot SDK
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+      - name: Update the Installed Python
+        run: apt-get update && apt-get install -y python3-pip python3
+      - name: Building the dependencies from the Polkadot SDK
+        run: forklift cargo build --locked --profile production -p pallet-revive-eth-rpc -p revive-dev-node
```

Comment:
use release build for CI it's faster to build

Reply:
we already build that in evm-test-suite, not too familiar with CI here but I wonder if we can share the step, or make the retester run an extra step in the evm-test-suite